tests/generation/test_utils.py

```diff
@@ -1328,9 +1253,8 @@ def test_dola_decoding_sample(self):
                 "use_cache": hasattr(config, "use_cache"),  # Some models don't support the cache
             }
             generation_kwargs.update({"dola_layers": "low"})
-            model_kwargs = {"attention_mask": attention_mask} if attention_mask is not None else {}
-            output_dola = model.generate(input_ids, **model_kwargs, **generation_kwargs, **inputs_dict)
-            self._check_outputs(output_dola, input_ids, model.config, use_cache=hasattr(config, "use_cache"))
+            output_dola = model.generate(**generation_kwargs, **inputs_dict)
+            self._check_outputs(output_dola, main_input, model.config, use_cache=hasattr(config, "use_cache"))
```

Comment:
totally unrelated to this PR: why is it `hasattr(config, "use_cache")` and not `getattr`?

Reply:
👀 good catch, will change